lib/perl/Genome/Sys/Lock/NessyBackend.pm

```diff
@@ -178,6 +181,32 @@ sub min_timeout {
     return 5;
 }
 
+sub _translate_resource {
```

Comment:
Ah, I didn't think that `Path::Class` was installed -- I should have checked.  If `Path::Class` is preferred, I'm happy to update!